doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,154 @@
+** `rhpam-production`: An environment for running existing services for staging and production purposes. This environment includes {CENTRAL} Monitoring, Smart Router, and two groups of {KIE_SERVER} pods; you can deploy and undeploy services on every such group and also scale the group up or down as necessary. Use {CENTRAL} Monitoring to deploy, run, and stop the services and to monitor their execution.
+** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. This environment includes {CENTRAL} Monitoring and Smart Router; you can configure one or more {KIE_SERVER} replicated pods that build a service from source. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment; you can use any container-based integration workflows and do not need to use any other tools to manage the pods. 
+** `rhpam-authoring`: An environment for creating and modifying services using {CENTRAL}. It consists of pods that provide {CENTRAL} for the authoring work and a {KIE_SERVER} for test execution of the services.
+** `rhpam-authoring-ha`: An environment for creating and modifying services using {CENTRAL}. It consists of pods that provide {CENTRAL} for the authoring work and a {KIE_SERVER} for test execution of the services. This version of the authoring environment supports scaling the {CENTRAL} pod to ensure high availability.
++
+. Add lines to the YAML source to modify the environment, using the snippets listed in this document as a reference:
+** The following snippet adds an immutable {KIE_SERVER} pod that builds a service from source. You must add at least one copy of this snippet when creating an immutable environment.
++
+[subs="attributes,verbatim,macros"]
+----
+  objects:
+    servers:
+      - build:
+          kieServerContainerDeployment: <deployment>
+          gitSource:
+            uri: <url>
+            reference: <branch>
+            contextDir: <directory>
+----
++
+Replace the following values:
++
+*** `<deployment>`: The identifying information of the decision service (KJAR file) that is built from your source. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. The Maven build process must produce all these files from the source in the Git repository.
```

Comment:
"The format is `<container...`" (no colon, per IBM, or make complete sentence)  Probably same at "separator, for example, `container...`. This would also be consistent with what you've done elsewhere in the doc.